alibi/explainers/tests/test_integrated_gradients.py

```diff
@@ -420,13 +425,14 @@ def test_integrated_gradients_binary_classification_layer(ffn_model, method, lay
 
     explanations = ig.explain(X_test,
                               baselines=baselines,
-                              target=test_labels)
+                              target=test_labels,
+                              compute_layer_inputs_gradients=layer_inputs_attributions)
 
     assert isinstance(explanations, Explanation)
-    if layer is not None:
+    if layer is not None and not layer_inputs_attributions:
         layer_out = layer(X_test).numpy()
         assert explanations['data']['attributions'][0].shape == layer_out.shape
-    else:
+    elif layer is None:
         assert explanations['data']['attributions'][0].shape == X_test.shape
```

Comment:
Looks like there are no assertions performed when `layer_input_attributions=True`, is there something we can add or is that not possible? E.g. call `layer(X_test)` and then check `layer.input_shape`?